integration-tests/src/test/resources/only-resources-file/workload/build.gradle.kts

```
@@ -0,0 +1,23 @@
+plugins {
+    kotlin("multiplatform")
+    id("com.google.devtools.ksp")
+}
+
+version = "1.0-SNAPSHOT"
+
+kotlin {
+    jvm {
+        withJava()
+    }
+}
+
+dependencies {
+    add("kspMetadata", project(":test-processor"))
+}
+
+// workaround for https://github.com/google/ksp/issues/567
+tasks.withType<org.jetbrains.kotlin.gradle.dsl.KotlinCompile<*>>().all {
+    if (name != "kspKotlinMetadata") {
+        dependsOn("kspKotlinMetadata")
+    }
+}
```

I think this workaround is no longer needed.

```suggestion
dependencies {
    add("kspCommonMainMetadata", project(":test-processor"))
}
```